### PR TITLE
[B2BP-249] Add styling for Hero

### PR DESCRIPTION
**Pull request by Meolocious**
Depends on #99 
Depends on #83 

#### List of Changes
- Updated Hero.tsx to address style issues.
- Modified layout.tsx to fix font styles.
- Added markdown rendering

#### Motivation and Context
This change is necessary to address style-related issues in the Hero and layout components. The Hero.tsx file was modified to fix the styling problem, and adjustments were made in the layout.tsx file to ensure consistent font styles throughout the application.

#### How Has This Been Tested?
The changes have been tested thoroughly to ensure that the style issues are resolved and that there are no unintended consequences. Testing included verification of the appearance and behavior of affected components in various scenarios. Specific test cases included [describe any specific test scenarios or steps performed].

#### Types of changes
- [ ] Chore (nothing changes from a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Doesn't need any documentation changes.

### CODE REVIEW

**Review by datalek — CHANGES_REQUESTED** [review]
To be adapted according to what has been done for https://github.com/pagopa/b2b-portals/pull/83.

**Review by datalek — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    // Not using isValidMuiIcon because typescript doesn't register iconName as non-null if we do
+    return null;
+  }
+
+  // Convert from camel case (anExample) or pascal case (AnExample) to snake case (an_example)
+  return iconName
+    .replace(/(?:^|\.?)([A-Z])/g, (_, y) => `_${y.toLowerCase()}`)
+    .split(/^_/)
+    .join('');
+};
```
[the remark this answers is not in the document]
Comment (simosalsi) [reply apps/nextjs-website/src/utils.ts:18]: This is used as of the latest commit

File: apps/nextjs-website/src/utils.ts
Comment (datalek) [inline apps/nextjs-website/src/utils.ts:1]: File named `util`, `common`, or `misc` provide no sense of what the file contains. This makes it harder for devs to use the file and makes it harder for maintainers to keep the file focused. 
I would rename it to `icons.ts`, or place it inside the `components` folder, within an `Icons` subfolder, in an index.ts file (`components/Icons/index.ts`).

**Review by datalek — APPROVED** [review]

**Review by datalek — APPROVED** [review]
LGTM

**Pull request merged**